modules/fts/6_6_content/fts-supported-queries-compound-query.adoc

```diff
@@ -0,0 +1,9 @@
+[#compound-queries]
```

Comment:
There's no real need for anchors at the top of the page.

Reply:
removed anchor